plugins/modules/inventory_intent.py

```diff
@@ -1109,6 +1110,41 @@ def export_device_details(self):
 
         return self
 
+    def check_ap_devices(self, device_ips):
```

Comment:
as we are getting ap_devices, can we rename this to get_ap_devices?

Reply:
rename the function.